frame/bags-list/remote-tests/Cargo.toml

```diff
@@ -31,7 +31,5 @@ sp-std = { path = "../../../primitives/std", version = "4.0.0" }
 remote-externalities = { path = "../../../utils/frame/remote-externalities", version = "0.10.0-dev" }
 
 # others
-tokio = { version = "1", features = ["macros"] }
 log = "0.4.14"
-structopt = "0.3.25"
-clap = "2.34.0"
```

Comment:
I guess clap wasn't used at all by this crate?

Reply:
yes